storage-account-rd-data-extracts.tf

```diff
@@ -2,6 +2,22 @@ locals {
   rd_data_extract_product         = "rddataextract"
   rd_data_extract_account_name    = join("", [local.rd_data_extract_product, var.env])
   rd_data_extract_container_name  = "rd-data-extract"
+
+  allowed_roles = [
+    "Storage Blob Data Contributor",
+    "Storage Blob Delegator",
+    "Storage Blob Data Reader"
+  ]
+}
+
+data "azuread_group" "sc_group" {
+  display_name     = "DTS Ref Data SC (env:production)"
```

Comment:
we don't generally put an env in a team group like this but it should be fine, I can see it's created, (although there's no one in there)

Reply:
I will remove this env variable and create another PR.